maestro-utils/src/main/kotlin/Insight.kt

```diff
@@ -16,16 +16,20 @@ object Insights {
 
     fun clear() {
         insights.clear()
+        onInsightsUpdated(insights.toList())
     }
 
     fun onInsightsUpdated(callback: (List<Insight>) -> Unit) {
```

Comment:
This only allows a single listener at a time

Reply:
Updated to list of listeners ✅